pkg/META.in

```diff
@@ -10,7 +10,6 @@ archive(syntax, preprocessor, utf8) = "-ignore"
 preprocessor = "ocamlreason_preprocessor_where_does_this_end_up"
 
 # Toplevel
-archive(byte, toploop) = "@compiler-libs/ocamlcommon.cma @str/str.cma"
```

Comment:
I'm not positive that this line can be removed. Everything still builds fine, but it looks like it might be necessary for toplevel/retop itself?